.drone.jsonnet

```diff
@@ -30,6 +30,14 @@
     ],
   },
 
+  local release(version) = std.mergePatch(
+    golang(version) { commands: ['curl -sL https://git.io/goreleaser | bash'] },
+    ({
+      environment: { GITHUB_TOKEN: { from_secret: 'github_token' } },
+      when: { event: 'tag' },
+    })
+  ),
```

Comment:
`std.mergePatch` is not required to be used manually in Jsonnet. Instead, you can use syntax features:

```jsonnet
  local release(version) = golang(version) {
    commands: ['curl -sL https://git.io/goreleaser | bash'] },
    environment+: { GITHUB_TOKEN: { from_secret: 'github_token' } },
    when: { event: 'tag' },
  },
```

Notice the use of `+:` (merge) and `:` (override)

Reply:
Ah damn, I always wondered what the `+:` syntax meant. I will fix in next commit.

Reply:
https://tanka.dev/jsonnet/overview#merging

Reply:
@sh0rez Nice! Pushed just now.